src/etc/inc/services.inc

```diff
@@ -2316,10 +2316,17 @@ trap := "{$config['snmpd']['trapstring']}"
 
 EOD;
 		}
-
-		$sysDescr = "{$g['product_name']} " . php_uname("n") .
-			" {$g['product_version']} {$g['platform']} " . php_uname("s") .
-			" " . php_uname("r") . " " . php_uname("m");
+		
+		if (!empty($g['product_version_string'])) {
```

Comment:
`$g['product_version_string']` is always defined and not empty.  Consider always using it